### PR TITLE
chore: Restore installing image puller on openshift using cli

### PR DESCRIPTION
**Pull request by tolusha**
## What does this pull request change?
chore: Restore installing image puller on openshift using cli

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/CRW-5723

## Specify the version of the product this pull request applies to
7.84.x

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by dmytro-ndp — APPROVED** [review]

File: modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,159 @@
+:_content-type: PROCEDURE
+:description: Installing {image-puller-name-short} on OpenShift using CLI
+:keywords: administration-guide, image-puller, openshift, openshift-templates, CLI
+:navtitle: Installing {image-puller-name-short} on OpenShift using CLI
+:page-aliases: .:installing-image-puller-on-openshift-using-openshift-templates.adoc,installing-image-puller-on-openshift-using-openshift-templates.adoc
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc]:
```suggestion
//:page-aliases: .:installing-image-puller-on-openshift-using-openshift-templates.adoc,installing-image-puller-on-openshift-using-openshift-templates.adoc
```

File: modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+.Procedure
+
+. Gather a list of relevant container images to pull by navigating to the links:
+* `pass:c,a,q[{prod-url}]/plugin-registry/v3/external_images.txt`
+* `pass:c,a,q[{prod-url}]/devfile-registry/devfiles/external_images.txt`
+
+. Define the memory requests and limits parameters to ensure pulled containers and the platform have enough memory to run.
++
+To define the minimal value for `CACHING_MEMORY_REQUEST` or `CACHING_MEMORY_LIMIT`, consider the necessary amount of memory required to run each of the container images to pull.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc]:
```suggestion
When defining the minimal value for `CACHING_MEMORY_REQUEST` or `CACHING_MEMORY_LIMIT`, consider the necessary amount of memory required to run each of the container images to pull.
```

File: modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
++
+To define the maximal value for `CACHING_MEMORY_REQUEST` or `CACHING_MEMORY_LIMIT`, consider the total memory allocated to the DaemonSet Pods in the cluster:
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc]:
```suggestion
When defining the maximal value for `CACHING_MEMORY_REQUEST` or `CACHING_MEMORY_LIMIT`, consider the total memory allocated to the DaemonSet Pods in the cluster:
```

File: modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+(memory limit) * (number of images) * (number of nodes in the cluster)
+----
++
+====
+Pulling 5 images on 20 nodes, with a container memory limit of `20Mi` requires `2000Mi` of memory.
+====
+
+. Clone the {image-puller-name-short} repository and get in the directory containing the OpenShift templates:
++
+[source,bash,subs="+attributes,+quotes"]
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc]: I recently learned that we're not supposed to use `bash` source language for terminal commands: https://redhat-documentation.github.io/supplementary-style-guide/#code-example-syntax-highlighting

Can you fix it, please? I noticed you using it a couple of times in the doc.

**Review by deerskindoll — COMMENTED** [review]
fix for the antora issue and a couple of suggestions

File: modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc [hunk 5]
```diff
@@ -0,0 +1,159 @@
+:_content-type: PROCEDURE
+:description: Installing {image-puller-name-short} on OpenShift using CLI
+:keywords: administration-guide, image-puller, openshift, openshift-templates, CLI
+:navtitle: Installing {image-puller-name-short} on OpenShift using CLI
+//:page-aliases: .:installing-image-puller-on-openshift-using-openshift-templates.adoc,installing-image-puller-on-openshift-using-openshift-templates.adoc
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc]:
```suggestion
// :page-aliases: .:installing-image-puller-on-openshift-using-openshift-templates.adoc,installing-image-puller-on-openshift-using-openshift-templates.adoc
```

**Review by deerskindoll — COMMENTED** [review]
forgot to add space, sorry

**Review by ibuziuk — APPROVED** [review]

**Comment by deerskindoll** [comment]
@tolusha I finally realized why the antora check is failing. you forgot to update nav.adoc with the new doc. the build won't pass until you do that.

**Pull request merged**